_posts/2021-04-20-custom-errors.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+published: true
+title:  Custom Errors in Solidity
+date:   2021-04-20
+author: Solidity Team
+category: Explainers
+---
+
+[Solidity v0.8.4](https://github.com/ethereum/solidity/releases/tag/v0.8.4) provides a convenient
+and gas-efficient way to explain to user why an operation failed. Errors can be defined inside and
+outside of contracts (including interfaces and libraries).
+
+## Example
+
+The following contract shows an example usage of an Error:
+
+```solidity
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.4;
+
+contract VendingMachine {
+    address payable owner = payable(msg.sender);
+    error Unauthorized();
+    function withdraw() public {
+        if (msg.sender != owner)
+            revert Unauthorized();
+
+        owner.transfer(address(this).balance);
+    }
+    // ...
+}
+```
+
+The syntax of Errors are similar to that of
```

Comment:
Why capitalize Errors here? I'd just say ``errors``. And is syntax not singular? (not sure, but for me it sounds more right)

```suggestion
The syntax of errors is similar to that of
```